src/Bicep.Core/Parser/Lexer.cs

```diff
@@ -214,7 +214,7 @@ private IEnumerable<SyntaxTrivia> ScanLeadingTrivia()
         private void LexToken()
         {
             textWindow.Reset();
-            ScanLeadingTrivia();
```

Comment:
`ScanLeadingTrivia` uses `yield return` inside. The return value wasn't enumerated, so nothing actually happened by calling it here.

Reply:
Nice!